src/test/resources/tf/Azure-Resource-Group-Does-Not-Have-A-Resource-Lock/Azure-Resource-Group-Does-Not-Have-A-Resource-Lock-positive-1.json

```diff
@@ -0,0 +1,31 @@
+{
+    "resource": [
+        {
+            "azurerm_resource_group": [
+                {
+                    "example": [
+                        {
+                            "location": "West Europe",
+                            "name": "locked-resource-group"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "azurerm_public_ip": [
```

Comment:
There is no "azurerm_management_lock" block, so this should be a negative policy.